src/app.module.ts

```diff
@@ -1,11 +1,26 @@
 import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
-import { MensajesController } from './mensajes/mensajes.controller';
+import { Message } from './messages/entities/message.entity';
+import { MessagesController } from './messages/messages.controller';
+import { MessagesService } from './messages/messages.service';
 
 @Module({
-  imports: [],
-  controllers: [AppController, MensajesController],
-  providers: [AppService],
+  imports: [
+    TypeOrmModule.forRoot({
+      type: 'mysql',
+      host: 'localhost',
+      port: 3306,
```

Comment:
create an .env file and use `PORT`, `HOST`, `DB_USERNAME`, `DB_PASSWORD` and `DB_NAME` from file

you can check what's the purpose of a .env file here: https://docs.nestjs.com/techniques/configuration